EDDiscovery/UserControls/Overlays/UserControlSurveyor.cs

```diff
@@ -417,15 +417,15 @@ private string InfoLine(ISystem sys, StarScan.ScanNode sn, JournalScan js)
                 information.Append(" (Mapped & Discovered)".T(EDTx.UserControlSurveyor_MandD));
                 if (showValuesToolStripMenuItem.Checked)
                 {
-                    information.Append(' ').Append(ev.EstimatedValueBase.ToString("N0")).Append(" cr");
```

Comment:
Umm its saying what you'll get for it, if its already been mapped/discovered. Base is right